03_launch_mgmt_cluster.sh

```diff
@@ -236,13 +236,33 @@ function make_bm_hosts() {
   done
 }
 
+#
+# Create each bmh and secret in a separated file
+# TODO (mboukhalfa): removed the old make_bm_hosts when this adopted by all tests 
+#
+function make_bm_hosts_seprated() {
```

Comment:
IMO the name `make_separated_bm_hosts()` might be more suitable. Anw, I think it's better to add a custom param `separate=${1:-false}`, instead of duplicating the function

Reply:
Please check now in once function and saving in separated files and everything in the old file